scripts/generateConjure.sh

```diff
@@ -25,13 +25,33 @@ function generateConjure() {
     printf "Fetching conjure IR\n  Coordinate: %s\n  Path: %s\n" "$GROUP_ID:$ARTIFACT_ID" "$OUT_DIR"
     "$SCRIPT_DIR/getConjureIr.sh" "$GROUP_ID" "$ARTIFACT_ID"
     echo "  - Generating typescript"
-    $CONJURE_LITE generate --ir "${SCRIPT_DIR}/../tmp/${ARTIFACT_ID}.conjure.json" --outDir "$OUT_DIR"
+    $CONJURE_LITE generate --ir "${SCRIPT_DIR}/../tmp/${ARTIFACT_ID}.conjure.json" --outDir "$OUT_DIR" --header "/**/"
 
     formatTypescript "$OUT_DIR" "$PACKAGE_PATH"
 }
 
-generateConjure "com.palantir.artifacts" "artifacts-sites-api" "${SCRIPT_DIR}/../packages/cli"
-
 generateConjure "com.palantir.object-set-service" "object-set-service-api" "${SCRIPT_DIR}/../packages/client"
 generateConjure "com.palantir.object-set-watcher" "object-set-watcher-api" "${SCRIPT_DIR}/../packages/client"
 generateConjure "com.palantir.ontology" "ontology-metadata-api" "${SCRIPT_DIR}/../packages/client"
+
+# Reset git changes if the generated files are only changed by copyright year
+git status --porcelain --untracked-files=no | while read line; do
+    status=$(echo "$line" | awk '{print $1 }')
+    file=$(echo "$line" | awk '{print $2 }')
+
+    if [[ "$status" == "D" ]]; then
+        # do nothing
+        bnitgub="da"
+    else 
+        v=$(git diff -U0 $file | tail +5 | md5sum)
+        expected="ce0376e23ce2aa3d0dcd56c2befc2993  -" 
```

Comment:
So this is the hash we'd expect if only the copyright year changes?

Reply:
Yes